frontend/static/js/components/formComponents/selectPlayersForm.js

```diff
@@ -0,0 +1,64 @@
+import AForm from './AForm.js';
+import PlayerCard from './PlayerCard.js';
+
+export default class selectPlayersForm extends AForm {
+	constructor(TournamentData) {
+		super();
+		this.numberOfPlayers = TournamentData.player_amount;
+		this.tournamentName = TournamentData.name;
+		this.playerIds = TournamentData.player_ids;
+		this.generateForm = this.generateForm.bind(this);
+	}
+
+	generateForm() {
+		const form = document.createElement('form');
+		form.classList.add('player-selection');
+
+		const playerList = document.createElement('section');
+		playerList.classList.add('player-list', `player-${this.numberOfPlayers}`);
+		form.appendChild(playerList);
+
+		for (let i = 0; i < this.numberOfPlayers; i++) {
+			if (i == 0) {
+				const playerCard = PlayerCard(
+					localStorage.getItem('username'),
+					// localStorage.getItem('userImage')
+					'../static/assets/img/default-user.png',
```

Comment:
if my PR happens to go thourgh, maybe would make sense to use the getFriendProfilePicture here, will give out the img attirubte to be appended to the card!

Reply:
I'll wait then 😎